var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -88,7 +91,9 @@ class Gdal(AutotoolsPackage):
     variant('cryptopp',  default=False, description='Include cryptopp support')
     variant('crypto',    default=False, description='Include crypto (from openssl) support')
 
-    extends('perl', when='+perl')
+    # FIXME: Allow packages to extend multiple packages
+    # See https://github.com/spack/spack/issues/987
+    # extends('perl', when='+perl')
     extends('python', when='+python')
```

Comment:
Without this, the module file generated by Spack doesn't set `PYTHONPATH`.

I doubt anyone needs the Perl bindings anyway.